app/templates/gentelella/guest/ticketing/pdf/ticket.html

```diff
@@ -28,7 +26,6 @@
 
```

Comment:
This doesn't seem to be changing anything so keeping it static is all good.
We can remove it too, will have to adjust the alignments.